### ISSUE TITLE
Arcade crashes on import in Python v3.8.0 - Could not find module 'avutil-56'

### ISSUE THREAD

**Issue by alex-gale** [pvcraven/arcade#514]
## Bug Report
When importing `arcade`, I get an error informing me that the module 'avutil-56' could not be found.

```
Unexpected error loading library avutil-56: Could not find module 'avutil-56'. Try using the full path with constructor syntax.
Unable to load the ffmpeg library.  Could not find module 'avutil-56'. Try using the full path with constructor syntax.
Unexpected error loading library avutil-56: Could not find module 'avutil-56'. Try using the full path with constructor syntax.
Traceback (most recent call last):
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\__init__.py", line 353, in __getattr__
    return getattr(self._module, name)
AttributeError: 'NoneType' object has no attribute 'Source'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 1, in <module>
    import arcade, enum
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\arcade\__init__.py", line 50, in <module>
    from arcade.sound import *
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\arcade\sound.py", line 81, in <module>
    def stop_sound(sound: pyglet.media.Source):
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\__init__.py", line 359, in __getattr__
    __import__(import_name)
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\__init__.py", line 146, in <module>
    add_default_media_codecs()
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\__init__.py", line 164, in add_default_media_codecs
    if have_ffmpeg():
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\__init__.py", line 180, in have_ffmpeg
    from . import ffmpeg_lib
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\ffmpeg_lib\__init__.py", line 36, in <module>
    from .libavcodec import *
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\ffmpeg_lib\libavcodec.py", line 45, in <module>
    from . import libavutil
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\ffmpeg_lib\libavutil.py", line 46, in <module>
    avutil = pyglet.lib.load_library(
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\lib.py", line 148, in load_library
    lib = ctypes.cdll.LoadLibrary(name)
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\ctypes\__init__.py", line 447, in LoadLibrary
    return self._dlltype(name)
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\ctypes\__init__.py", line 369, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'avutil-56'. Try using the full path with constructor syntax.
Unexpected error loading library avutil-56: Could not find module 'avutil-56'. Try using the full path with constructor syntax.
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\__init__.py", line 146, in <module>
    add_default_media_codecs()
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\__init__.py", line 164, in add_default_media_codecs
    if have_ffmpeg():
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\__init__.py", line 180, in have_ffmpeg
    from . import ffmpeg_lib
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\ffmpeg_lib\__init__.py", line 36, in <module>
    from .libavcodec import *
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\ffmpeg_lib\libavcodec.py", line 45, in <module>
    from . import libavutil
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\media\codecs\ffmpeg_lib\libavutil.py", line 46, in <module>
    avutil = pyglet.lib.load_library(
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\site-packages\pyglet\lib.py", line 148, in load_library
    lib = ctypes.cdll.LoadLibrary(name)
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\ctypes\__init__.py", line 447, in LoadLibrary
    return self._dlltype(name)
  File "C:\Users\alexg\AppData\Local\Programs\Python\Python38\lib\ctypes\__init__.py", line 369, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'avutil-56'. Try using the full path with constructor syntax.
```

### Actual behavior:
Arcade crashes when imported.

### Expected behavior:
Arcade is imported and the game runs.

### Steps to reproduce/example code:
- I upgraded to Python 3.8.0 from Python 3.6.0
- I installed `arcade` globally using `pip install arcade` as an administrator
- I ran my game, which threw the above error at `import arcade`

**Comment by pvcraven** [comment]
Ok, finally able to replicate this.

**Comment by pvcraven** [comment]
See related issue just created in Pyglet:

https://github.com/pyglet/pyglet/issues/90

**Comment by alex-gale** [comment]
Cheers! If it helps at all, when trying to compile with PyInstaller in Python 3.6, I got a very similar error when running the .exe regarding `ctypes`, ffmpeg and the `avutil-56` module. Might be related?

**Comment by pvcraven** [comment]
Should be fixed with the next Pyglet maintenance release. See https://github.com/pyglet/pyglet/issues/90

**Comment by dmcdrmtt** [comment]
Regardless of how you set up your libraries or project root structures this missing file error seems to occur. I got around it by starting my code with this code structure: (also note that two other files will create the same concern, hence the three lines below)

# Game code starting template

from ctypes import *
avcodec = cdll.LoadLibrary('c:\\python38\\lib\\site-packages\\pyglet_ffmpeg\\win64\\avcodec-58.dll')
avformat = cdll.LoadLibrary("c:\\python38\\lib\\site-packages\\pyglet_ffmpeg\\win64\\avformat-58.dll")
swscale = cdll.LoadLibrary("c:\\python38\\lib\\site-packages\\pyglet_ffmpeg\\win64\\swscale-5.dll")

import arcade

PEP 8 will complain that your module import command is not at the top of the file but it won't affect anything

**Comment by pvcraven** [comment]
Fixed with 2.2.0 release forcing pyglet 1.4.8
